balancer/weightedtarget/weightedtarget_test.go

```diff
@@ -568,8 +575,12 @@ func (s) TestWeightedTarget_TwoSubBalancers_MoreBackends(t *testing.T) {
 		t.Fatalf("want %v, got %v", want, err)
 	}
 
+	scConnErr := errors.New("this is definitely a connection error. please check your wiring")
```

Comment:
Nit: could you please use a simpler error message. something like `subConn connection error`. And move this definition to be after the comment on line 580. So, it would be:

```golang
	// Turn sc1's connection down.
	scConnErr := errors.New("subConn connection error")
	wtb.UpdateSubConnState(sc1, balancer.SubConnState{
		ConnectivityState: connectivity.TransientFailure,
		ConnectionError:   scConnErr,
	})
```

Although the Go style guide doesn't differentiate between:
```golang
	wtb.UpdateSubConnState(sc1, balancer.SubConnState{
		ConnectivityState: connectivity.TransientFailure,
		ConnectionError:   scConnErr})
```
and
```golang
	wtb.UpdateSubConnState(sc1, balancer.SubConnState{
		ConnectivityState: connectivity.TransientFailure,
		ConnectionError:   scConnErr,
	})
```
I have seen the latter form used much more widely and I would recommend it as well.

Reply:
sure. ill make this update. I didnt look at how the styling looks like in other places for this one.